Review/delivery/http/review_delivery_test.go

```diff
@@ -19,7 +19,13 @@ import (
 	"github.com/stretchr/testify/mock"
 )
 
-func TestReviewHandler_AddReview(t *testing.T) {
+const stardomainreview = "*domain.Review"
```

Comment:
please use camel case for these private constant variables to follow the golang naming convention